pom.xml

```diff
@@ -69,7 +69,7 @@
     <slf4j.version>1.7.21</slf4j.version>
     <tomcat.version>8.5.11</tomcat.version>
     <elasticsearch.version>2.4.4</elasticsearch.version>
-    <orchestrator.version>3.15.0.1038</orchestrator.version>
+    <orchestrator.version>3.15-SNAPSHOT</orchestrator.version>
```

Comment:
Yes I've just set this to be able to use the MsSQL automatic collation. I'll update this.